samples/multi-bindings/README.md

```diff
@@ -0,0 +1,59 @@
+# Multi-binding example
+
+This example showcases the interplay of using a megazord build of a Rust library, the Glean RLB and glean-ffi,
```

Comment:
Can we linkify megazord to point at some repo?

Reply:
Good point.